deploy/docker/Dockerfile

```diff
@@ -1,13 +1,11 @@
 FROM ubuntu:14.04
 
-ENV http_proxy='http://172.28.40.9:3128/'
-ENV https_proxy='http://172.28.40.9:3128/'
-ENV ftp_proxy='http://172.28.40.9:3128/'
-ENV no_proxy='127.0.0.1,localhost'
+#ENV http_proxy='http://172.28.40.9:3128/'
```

Comment:
I think we should define a `PROXY` env var here, which should be the only point where the user may specify a proxy or leave it empty.

Then, this variable should be used in: 
- lines 4,5,7,8, and as an argument for provision.sh in line 17